cmd/src/bin/tikv-server.rs

```diff
@@ -136,6 +136,11 @@ fn main() {
                      leaves it empty will disable Prometheus push",
                 ),
         )
+        .arg(
+            Arg::with_name("dy-config")
```

Comment:
```suggestion
            Arg::with_name("dynamic-config")
```

We don't need abbreviations.